### ISSUE TITLE
Loss = Nan

### ISSUE THREAD

**Issue by Fiona2016Xue** [UpCoder/centerloss#2]
你好，我按照您的方法，在softmax_loss基础上，增加了center loss, 但是会出现loss = nan的情况，从而运行不了了，这是什么问题呢
